.cirrus.yml

```diff
@@ -308,21 +320,27 @@ alt_build_task:
     matrix:
       - env:
             ALT_NAME: 'Build Each Commit'
+            CACHE_KEY: 'alteach-${CIRRUS_BUILD_ID}'
       - env:
             ALT_NAME: 'Windows Cross'
+            CACHE_KEY: 'altwin-${CIRRUS_BUILD_ID}'
       - env:
             ALT_NAME: 'Build Without CGO'
+            CACHE_KEY: 'altcgo-${CIRRUS_BUILD_ID}'
       - env:
             ALT_NAME: 'Test build RPM'
+            CACHE_KEY: 'altrpm-${CIRRUS_BUILD_ID}'
       - env:
             ALT_NAME: 'Alt Arch. Cross'
-    # This task cannot make use of the shared repo.tbz artifact.
+            CACHE_KEY: 'altcross-${CIRRUS_BUILD_ID}'
+    # This task performs alternate/cross compiles, therefore it
+    # cannot make use of the repo cache from the 'build' task.
     clone_script: *full_clone
     setup_script: *setup
     main_script: *main
-    # Produce a new repo.tbz artifact for consumption by 'artifacts' task.
-    repo_prep_script: *repo_prep
-    repo_artifacts: *repo_artifacts
+    # Produce new repo cache based on a special $CACHE_KEY for consumption
+    # by 'artifacts' task.
+    cache_upload_script: *cache_upload
```

Comment:
This is probably my inability to read convoluted YAML, but as best I can tell `cache_upload` requires `DISTRO_NV`, which does not seem to be set here? Also, CI is still broken